### PR TITLE
add guidance toggle checkbox

### LINKED ISSUE #1541: Add toggle guidance
Adding Toggle guidance and when to use vs checkbox. Is it only for immediate changes that become visible on the page?
Do we recommend text on the Right vs left of the toggle?

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
Adds guidance for checkbox toggle 
this comes from a discussion with Edgar 

#### What are the relevant issues?
closes #1541

### CODE REVIEW

File: aries-site/src/pages/components/checkbox.mdx [hunk 1]
```diff
@@ -34,6 +32,14 @@ There are three states which CheckBox provides:
 
 Checked and unchecked states refer to when an individual CheckBox is checked or when all options from a group are checked.
 
+### Toggle
+
+- [Toggle](#toggle) is a single option which prompts users to choose between two mutually exclusive options.
+- Toggle is prefered to quickly switch between two possible states. Toggles usually provide immediate results (i.e. no form submission is required) in which gives users the ability to control their preference.
```
Comment (taysea) [inline aries-site/src/pages/components/checkbox.mdx:38]:
```suggestion
- Toggle is preferred to quickly switch between two possible states. Toggles usually provide immediate results (i.e. no form submission is required) in which gives users the ability to control their preference.
```

File: aries-site/src/pages/components/checkbox.mdx [hunk 2]
```diff
@@ -34,6 +32,14 @@ There are three states which CheckBox provides:
...
+- When using toggles the order of text and toggle can be controlled using the [reverse prop](https://v2.grommet.io/checkbox?theme=hpe#reverse), if the toggles
```
Comment (taysea) [inline aries-site/src/pages/components/checkbox.mdx:39]:
```suggestion
- When using toggles the order of text and toggle can be controlled using the [reverse prop](https://v2.grommet.io/checkbox?theme=hpe#reverse). If the toggles 
```

File: aries-site/src/pages/components/checkbox.mdx [hunk 3]
```diff
@@ -34,6 +32,14 @@ There are three states which CheckBox provides:
 
 Checked and unchecked states refer to when an individual CheckBox is checked or when all options from a group are checked.
 
+### Toggle
+
+- [Toggle](#toggle) is a single option which prompts users to choose between two mutually exclusive options.
+- Toggle is prefered to quickly switch between two possible states. Toggles usually provide immediate results (i.e. no form submission is required) in which gives users the ability to control their preference.
+- When using toggles the order of text and toggle can be controlled using the [reverse prop](https://v2.grommet.io/checkbox?theme=hpe#reverse), if the toggles 
+are used in a group that contains only toggles then it is prefered to have to text on the left and the toggle on the right.
```
Comment (taysea) [inline aries-site/src/pages/components/checkbox.mdx:40]:
```suggestion
are used in a group that contains only toggles then it is preferred to have the text on the left and the toggle on the right.
```

**Review by taysea — APPROVED** [review]
LGTM. just letting check pass.

**Comment by britt6612** [comment]
Thank you @taysea

**Pull request merged**